README.md

```diff
@@ -47,7 +47,7 @@ Several agent configuration options are exposed through this module.  Please fin
 
 | Name | Version |
 |------|---------|
-| <a name="provider_aws"></a> [aws](#provider\_aws) | 4.2.0 |
+| <a name="provider_aws"></a> [aws](#provider\_aws) | >= 3.72.0 |
```

Comment:
Does this need to cap at `< 4.0.0`? Double-checking since this change will accept a major version lower. ?